code/modules/art/paintings.dm

```diff
@@ -423,31 +501,20 @@
 	desc_with_canvas = "A painting hung away from lesser minds."
 	persistence_id = "library_private"
 
-/obj/structure/sign/painting/vv_get_dropdown()
-	. = ..()
-	VV_DROPDOWN_OPTION(VV_HK_REMOVE_PAINTING, "Remove Persistent Painting")
-
-/obj/structure/sign/painting/vv_do_topic(list/href_list)
+/// Simple painting utility.
+/obj/item/paint_palette
```

Comment:
This is only an idea but paint palettes should be able to store up to a dozen colors or so which then can be selected (or removed from the list for) through a radial menu. This sort of behavior would match more how they work in real life.

Reply:
Good idea, but i'd probably have to bind it with canvas tgui so it's faster than just changing color by hand.